warhorn/default_config.xml

```diff
@@ -3,8 +3,8 @@
 <!-- To read full documentation for this file, please refer to user_generated/config_sample.xml -->
 
 <data>
-    <warhorn name="Warhorn">
-        <dependency name="jira" install="no" user="no"/>
+    <warhorn name="Warhorn" dependency_filepath="Provide filepath, where version details for packages below are defined. By default the versions defined in requirements.txt at the base directory would be used">
```

Comment:
Hi @brettleehari,

The design for the dependency_filepath attribute was not mentioned anywhere in the design document.

Reply:
I shall explain this over the call. We had discussed about this item.